src/TesApi.Web/Storage/DefaultStorageAccessProvider.cs

```diff
@@ -167,15 +126,15 @@ public async Task<string> MapLocalPathToSasUrlAsync(string path, bool getContain
                         var policy = new SharedAccessBlobPolicy()
                         {
                             Permissions = SharedAccessBlobPermissions.Add | SharedAccessBlobPermissions.Create | SharedAccessBlobPermissions.List | SharedAccessBlobPermissions.Read | SharedAccessBlobPermissions.Write,
-                            SharedAccessExpiryTime = DateTime.Now.Add(sasTokenDuration)
+                            SharedAccessExpiryTime = DateTime.Now.Add(SasTokenDuration)
```

Comment:
Let's use DateTimeOffset.UtcNow here to stay consistent with UTC times throughout the code

https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.storage.blob.sharedaccessblobpolicy.sharedaccessexpirytime?view=azure-dotnet

Reply:
Good point. I will add an issue for this, as the existing implementation does not use UTC, and I didn't want to change it as part of this.